iothub/device/src/Transport/Mqtt/MqttTransportHandler.cs

```diff
@@ -995,18 +995,6 @@ private static object ConvertToSystemProperty(KeyValuePair<string, string> prope
                 return DateTime.ParseExact(property.Value, "o", CultureInfo.InvariantCulture);
             }
 
-            if (property.Key == MessageSystemPropertyNames.Ack)
```

Comment:
There was no reason for the device client side to look for this property as the service would never send it. This was the only use of DeliveryAcknowledgement, so we can safely delete that class

Reply:
Even a user wouldn't specify it?

Reply:
<img width="502" alt="image" src="https://user-images.githubusercontent.com/14119884/192026805-d9354b53-e139-4d31-90bf-2d1f199d014d.png">

Reply:
The service client message chooses this enum, yes. But the value isn't forwarded to the device/module recipient. The service just uses this value to decide under what circumstances the service enqueues a message feedback record for the service client to consume

Reply:
Ah